### DISCUSSION TITLE
Where are the Pino logs displayed?

### DISCUSSION THREAD

**Discussion by vmundhra** [BearStudio/start-ui-web#455] [Q&A] [answered]
Where can I view the server side logs? No logs are displayed in console. Where can we view and debug logs such as ctx.logger.info('Checking if the user exists'); that are in trpc routes.

**Comment by HugoPerard** [comment]
Hello @vmundhra ! 
You should find this logs on your console, that looks weird that you can't see them 🤔 
Can you check that your local env variable `LOGGER_LEVEL` is set at "info" ?

**Reply by vmundhra** [reply, marked as answer]
@HugoPerard Sorry my bad. My terminal is black, and I just realized the pino pretty options in logger.ts have color black for messages in code. I changed it to yellow and now it shows. Thanks alot.
